### ISSUE TITLE
two instances of accordian trigger each other

### ISSUE THREAD

**Issue by DrChills** [twbs/bootstrap#19806]
[(https://jsfiddle.net/DanielChilly/x7yqfomp/)]

Adding two instances of the Accordion, with correct ID changes, fails to act independantly from each other. The first will close all of the seconds accordion, and vica versa.

same jsfiddle tested with bootstrap 3 and works as expected.

**Comment by twbs-lmvtfy** [comment]
Hi @DrChills!

You appear to have posted a live example (https://fiddle.jshell.net/DanielChilly/x7yqfomp/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 108, column 13: [E014](https://github.com/twbs/bootlint/wiki/E014): Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by DrChills** [comment]
[(https://jsfiddle.net/DanielChilly/x7yqfomp/)]

**Comment by twbs-lmvtfy** [comment]
Hi @DrChills!

You appear to have posted a live example (https://fiddle.jshell.net/DanielChilly/x7yqfomp/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 108, column 13: [E014](https://github.com/twbs/bootlint/wiki/E014): Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by DrChills** [comment]
(https://jsfiddle.net/DanielChilly/x7yqfomp/)

**Comment by twbs-lmvtfy** [comment]
Hi @DrChills!

You appear to have posted a live example (https://fiddle.jshell.net/DanielChilly/x7yqfomp/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 108, column 13: [E014](https://github.com/twbs/bootlint/wiki/E014): Columns (`.col-*-*`) can only be children of `.row`s or `.form-group`s

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by DrChills** [comment]
https://jsfiddle.net/DanielChilly/uktuacak/

**Comment by Johann-S** [comment]
Duplicate of : https://github.com/twbs/bootstrap/issues/18824

and maybe one day this PR will be review : https://github.com/twbs/bootstrap/pull/18847
to fix this bug :smile:

**Comment by DrChills** [comment]
Ah Thanks!!! 

I've 'thumbed in' your change until it is fully working. I can confirm it solved the issue

**Comment by Johann-S** [comment]
Thank you for your feedbacks :+1:

**Comment by RyanZim** [comment]
@DrChills Since this is a duplicate, could you please close this?

**Comment by DrChills** [comment]
My bad!
